monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -165,11 +166,16 @@ object Rx {
   final case class Zip     [A, B]     (self: Rx[A], other: Rx[B])                   extends Rx[(A, B)]
   final case class DropRep [A]        (self: Rx[A])                                 extends Rx[A]
   final case class Merge   [A, B >: A](self: Rx[A], other: Rx[B])                   extends Rx[B]
```

Comment:
What do you think about remove `Merge` from here and implement it using `MergeN`?

Reply:
I don't know the exact performance implications, but since merging two Rx's is probably the most common situation in practice, it might be good to leave it in.